Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:lts-bullseye-slim AS base
```

Comment:
Hi @dcramer Just out of curiosity, why did you choose this Docker image of Node with this tag?

Reply:
Just chose an LTS version and went w/ slim to minimize the image. Lots of folks suggested using Google's lightweight images and/or chainguard, but the Docker image complexity gets intense fast. I do think it would be better to go w/ the Google approach as a v2 though.